pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -446,8 +447,28 @@ func (ctrl *Controller) syncStatusOnly(cfg *mcfgv1.ContainerRuntimeConfig, err e
 	return err
 }
 
+// addAnnotation adds the annotions for a ctrcfg object with the given annotationKey and annotationVal
+func (ctrl *Controller) addAnnotation(cfg *mcfgv1.ContainerRuntimeConfig, annotationKey, annotationVal string) error {
```

Comment:
what is an example of this key/value pair?

Reply:
An example is the new suffix annotation key I added
```
MCNameSuffixAnnotationKey = "machineconfiguration.openshift.io/mc-name-suffix"
``` 
and its value will be the suffix from the MC name.
I made the function generic in the even we would want to add more annotations in the future.